src/DependencyInjection/Configuration.php

```diff
@@ -41,6 +41,20 @@ public function getConfigTreeBuilder()
 
         $this->addBundlesSection($root);
 
+        $root
+            ->children()
+                ->arrayNode('ivory_ckeditor')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->enumNode('enabled')
+                            ->values([true, false, 'auto'])
+                            ->defaultValue('auto')
+                        ->end()
+                        ->scalarNode('config_name')->defaultValue('cmf_sonata_phpcr_admin_integration')->end()
```

Comment:
is this configuration automatically defined? if not, i would not use a default value at all, and make config_name required instead.